schemas/Path.json

```diff
@@ -120,14 +120,12 @@
 				"start": {
 					"description": "Ril100 des Startbahnhofs",
 					"type": "string",
-					"minLength": 2,
-					"maxLength": 10
```

Comment:
Da will jemand meine Datenbank sprengen. 👀

Reply:
Ich kann es auch auf 11 oder so setzen :D
Nur, es nervt irgendwann, das ständig erweitern zu müssen 😅

Im UK gibt es halt für manche Betriebsstellen nur den TIPLOC-Code, der 6 Zeichen lang ist, plus die 4 Byte lange Flagge. Ich kann das aber vielleicht noch durch eine eher "weiche" Prüfung im Checking-Script ersetzen. Da würden zum einen Flaggen als 2 Zeichen gerechnet, zum anderen könnte es bei sehr langen Codes eine Warnung ausgeben.

Deutsche Codes (also ohne Flagge) werden sowieso mit dem Betriebsstellenverzeichnis abgeglichen.

Reply:
Der Punkt ist, dass das Feld in der MySQL-Datenbank derzeit nur 10 Zeichen Platz hat. Mach mal auf 20 Zeichen, dass sollte erstmal reichen und ich kann die Datenbank entsprechend anpassen.